CONTRIBUTING.md

```diff
@@ -383,6 +383,87 @@ tested it becomes an officially supported Akka feature.
 
 [List of Akka features marked as may change](http://doc.akka.io/docs/akka/current/common/may-change.html)
 
+## Java APIs in Akka
+
+Akka, aims to keep 100% feature parity between the various language DSLs. Implementing even the API for Java in 
+Scala has proven the most viable way to do it, as long as you keep the following in mind:
+
+1. Keep entry points separated in `javadsl` and `scaladsl` unless changing existing APIs which for historical 
+   and binary compatibility reasons does not have this subdivision.
+   
+1. Have methods in the `javadsl` package delegate to the methods in the Scala API. The Akka Stream Scala instances 
```

Comment:
delegate to the methods in the Scala API, or to the common internal implementation